src/applications/personalization/dashboard-2/tests/e2e/mpi-error.cypress.spec.js

```diff
@@ -0,0 +1,23 @@
+import { mockFeatureToggles } from './helpers';
+
+import { mockLocalStorage } from '~/applications/personalization/dashboard/tests/e2e/dashboard-e2e-helpers';
+
+import mockMPIErrorUser from '~/applications/personalization/profile/tests/fixtures/users/user-mpi-error.json';
+
+describe('MyVA Dashboard', () => {
+  describe('when there is an error connecting to MPI', () => {
+    beforeEach(() => {
+      mockLocalStorage();
+
+      cy.login(mockMPIErrorUser);
```

Comment:
I believe this sole test case is sufficient - the mock error user includes both a 500 and 400 error response for the call to the MVI and EMIS service

```
"meta": {
    "errors": [
      {
        "externalService": "MVI",
        "startTime": "2020-08-21T22:17:28Z",
        "endTime": null,
        "description": "MVI_504, 504, Gateway timeout, Did not receive a timely response from an upstream server",
        "status": 504
      },
      {
        "externalService": "EMIS",
        "startTime": "2020-08-21T22:17:28Z",
        "endTime": null,
        "description": "EMISRedis::VeteranStatus::NotAuthorized, NOT_AUTHORIZED",
        "status": 401
      }
    ]
  }
  ```